lib/stoplight/default.rb

```diff
@@ -2,16 +2,15 @@
 
 module Stoplight
   module Default
-    WHITELISTED_ERRORS = [
-      NoMemoryError,
-      ScriptError,
-      SecurityError,
-      SignalException,
-      SystemExit,
-      SystemStackError
-    ].freeze
-
-    BLACKLISTED_ERRORS = [].freeze
+    ERROR_HANDLER = lambda do |error|
+      classes = [
+        Interrupt,
+        NoMemoryError,
+        SignalException,
+        SystemExit
+      ]
+      raise error if classes.any? { |klass| error.is_a?(klass) }
+    end
```

Comment:
Thanks for the feedback! 

1. I agree that this is not so nice. You can fall back to the default handler as the last line of your custom handler: `return Stoplight::Default::ERROR_HANDLER.call(error)`. That's a little wordy, so people probably won't do it.

2. I don't think there's any problem with catching and re-raising errors. 

3. The difference between my method and RSpec's method is cosmetic. 

4. I considered this approach, but ultimately decided against it because it could be confusing. Does `true` mean "re-raise the error", or is that `false`? By explicitly calling `raise` in your handler, there is no confusion. Also, the only thing Stoplight would ever do to handle an error is re-raise it, so why have that extra layer of indirection?

I think the first point is the biggest problem with this approach. There may be an easy way to fix it, like returning some sentinel value to use the default handler.